deployments/scripts/push-containers.sh

```diff
@@ -16,6 +16,7 @@ npmAuditVersion=$(docker run --rm huskyci/npmaudit:latest npm audit --version)
 yarnAuditVersion=$(docker run --rm huskyci/yarnaudit:latest yarn audit --version )
 safetyVersion=$(docker run --rm huskyci/safety:latest safety --version | awk -F " " '{print $3}')
 gitleaksVersion=$(docker run --rm huskyci/gitleaks:latest gitleaks --version)
+spotbugsVersion=$(docker run --rm huskyci/spotbugs:latest spotbugs -version)
```

Comment:
`spotbugs -version` is returning an error as `spotbugs` was not linked in any `$PATH`. What do you think of replacing it with `cat /opt/spotbugs/version`? 🙃